src/main/java/com/easyrest/components/menuPage/OrderConfirmation.java

```diff
@@ -0,0 +1,32 @@
+package com.easyrest.components.menuPage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class OrderConfirmation {
```

Comment:
I think in this class should be locator to field time picker and date picker and also methods that click on time picker and return component of time picker and  second method that click on date picker field and return the component of calendar

Reply:
Yes, but at this stage I cannot add them, and we do not need them for the test case.